docs/src/getting_started.md

```diff
@@ -39,6 +46,24 @@ pip3 install stata_kernel
 python3 -m stata_kernel.install
 ```
 
+### Jupyter
+
+If you chose to install Anaconda you already have Jupyter and Jupyter Lab installed.
+
+If you downloaded Miniconda, you need to install Jupyter and optionally Jupyter Lab:
+
+```bash
+conda install jupyter
+conda install jupyterlab
+```
+
+Otherwise you can install them using `pip`:
+
+```bash
+pip install jupyter
+pip install jupyterlab
+```
```

Comment:
I don't think you need to suggest installing `jupyter`, since that's just the metapackage. `jupyterlab` should install everything necessary.
`````suggestion
```bash
conda install jupyterlab
```

Otherwise you can install them using `pip`:

```bash
pip install jupyterlab
```
`````

Reply:
Oh sorry I accidentally wasn't looking at the most recent commit...